plugins/inventory/lxd.py

```diff
@@ -0,0 +1,1060 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Frank Dornheim <dornheim@posteo.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+    name: lxd
+    plugin_type: inventory
+    short_description: Returns Ansible inventory from lxd host
+    description:
+        - Get inventory from the lxd
+        - Uses a YAML configuration file that ends with 'lxd.(yml|yaml)'.
+    version_added: "2.1.0"
+    author: "Frank Dornheim (@conloos)"
+    options:
+        plugin:
+            description: token that ensures this is a source file for the 'lxd' plugin
+            required: true
+            choices: ['lxd']
+        url:
+            description:
+            - The unix domain socket path or the https URL for the LXD server.
+            - mostly: unix:/var/lib/lxd/unix.socket or unix:/var/snap/lxd/common/lxd/unix.socket
+            required: false
+            type: str
+        client_key:
+            description:
+            - The client certificate key file path.
+            required: false
+            aliases: [ key_file ]
+            default: $HOME/.config/lxc/client.key
+            type: path
+        client_cert:
+            description:
+            - The client certificate file path.
+            required: false
+            aliases: [ cert_file ]
+            default: $HOME/.config/lxc/client.crt
+            type: path
+        trust_password:
+            description:
+            - The client trusted password.
+            - You need to set this password on the LXD server before
```

Comment:
Here you use `LXD`, but you also use `lxd` a lot. What's the correct casing? Please use that everywhere.

Reply:
done